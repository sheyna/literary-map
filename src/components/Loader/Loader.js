import styles from './Loader.module.css';

// From Jonas Schmedtmann
function Loader() {
  return (
    <p className={styles.loader}>Loading...</p>
  )
}

export default Loader;
