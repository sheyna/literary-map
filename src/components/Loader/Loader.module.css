/* From Jonas Schmedtmann */
.loader {
  text-align: center;
  text-transform: uppercase;
  font-size: 2rem;
  font-weight: 600;
  margin: 4.8rem;
}
