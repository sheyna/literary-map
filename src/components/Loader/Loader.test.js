import ReactTestRenderer from 'react-test-renderer';
import Loader from  './Loader';

describe('Loader component', () => {
  it('should match snapshot', () => {
    const component = ReactTestRenderer.create(
      <Loader />
    );
    expect(component.toJSON()).toMatchSnapshot();
  });
});
