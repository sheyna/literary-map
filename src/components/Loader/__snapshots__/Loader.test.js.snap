// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component should match snapshot 1`] = `
<p
  className="loader"
>
  Loading...
</p>
`;
